run_service.sh

```diff
@@ -971,25 +988,26 @@ if [ "$local_service_hash" != "$remote_service_hash" ] || [ "$on_chain_agent_id"
           verify_staking_slots
 
           nft="bafybeig64atqaladigoc3ds4arltdu63wkdrk3gesjfvnfdmz35amv7faq"
-          export cmd=""
+          cmd="poetry run autonomy mint \
+              --retries $RPC_RETRIES \
+              --timeout $RPC_TIMEOUT_SECONDS \
+              --use-custom-chain \
+              service packages/valory/services/trader/ \
+              --key \"../$operator_pkey_path\" $password_argument \
+              --nft $nft \
+              -a $AGENT_ID \
+              -n $n_agents \
+              --threshold $n_agents \
+              --update \"$service_id\""
+
           if [ "${USE_STAKING}" = true ]; then
               cost_of_bonding=$MIN_STAKING_BOND_OLAS
-              poetry run python "../scripts/update_service.py" "../$operator_pkey_path" "$nft" "$AGENT_ID" "$service_id" "$CUSTOM_OLAS_ADDRESS" "$cost_of_bonding" "packages/valory/services/trader/" "$rpc" $password_argument
+              cmd+=" -c $cost_of_bonding --token $CUSTOM_OLAS_ADDRESS"
           else
               cost_of_bonding=$MIN_STAKING_BOND_XDAI
-              cmd="poetry run autonomy mint \
-                  --retries $RPC_RETRIES \
-                  --timeout $RPC_TIMEOUT_SECONDS \
-                  --use-custom-chain \
-                  service packages/valory/services/trader/ \
-                  --key \"../$operator_pkey_path\" $password_argument \
-                  --nft $nft \
-                  -a $AGENT_ID \
-                  -n $n_agents \
-                  -c $cost_of_bonding \
-                  --threshold $n_agents \
-                  --update \"$service_id\""
+              cmd+=" -c $cost_of_bonding"
```

Comment:
Use the already available autonomy command (`--token` option was not available and a custom script was created)